bindings/nodejs/generated.d.ts

```diff
@@ -633,6 +634,14 @@ export class Writer {
    */
   close(): Promise<void>
 }
+export class AsyncWriteWith {
+  apply(): Promise<void>
```

Comment:
Adding a new API called `apply` doesn't looks good to me. It's a goal for our nodejs binding to be looked like a native nodejs lib instead of following the same API pattern from rust core.

How about implement something like:

```js
op.write_with(path, content, { contentType: "application/json" }).await
```

Will it looks better from nodejs dev's view?

Reply:
This would be better from my side.

Reply:
Prefer this from me.